src/InAppUpdates.android.ts

```diff
@@ -90,11 +91,9 @@ export default class InAppUpdates extends InAppUpdatesBase {
     const { curVersion, toSemverConverter, customVersionComparator } =
       checkOptions || {};
 
-    if (!curVersion) {
-      this.throwError(
-        'You have to include at least the curVersion to the options passed in checkNeedsUpdate',
-        'checkNeedsUpdate'
-      );
+    let appVersion: string = getVersion();
+    if (curVersion) {
```

Comment:
Instead of always running `getVersion` (even if `curVersion` exists) could we only run it if it doesn't?

Reply:
sure why not, let me try